environment.yml

```diff
@@ -13,7 +13,6 @@ dependencies:
   - bzip2==1.0.6=1
   - cmake==3.9.1=0
   - cspice==66-0
-  - curl==7.55.1=0
```

Comment:
Is this not breaking? see #446

Reply:
I pretty sure this is a dependency of a dependency so we shouldn’t have to manage it separately. So if the version probably here is broken, removing the line has anaconda implicitly search for a version that isn’t broken but still satisfies the requirements. From using it personally (albeit briefly) didn’t seem to have any problem.

Reply:
This specific build of curl is broken. I think it would be better to remove the version number.

Reply:
I think it comes down to whether or not we use it directly, and if we care about the version number. With the answer to both of these is no, remove it altogether and allow another package to handle it itself. If we do use it directly and don’t care too much about version (which sounds like the case) then remove it and have no version number. I think we should stay away from build numbers from anything not built by us. The more flexible the environment file the better.

Reply:
@Kelvinrr that fix sounds good to me